crates/compiler/src/update/patch_module_group_graph/tests.rs

```diff
@@ -214,9 +212,7 @@ fn test_patch_module_group_graph_3() {
   let update_module_group_graph = module_group_graph_from_entries(
     &module_graph
       .entries
-      .clone()
-      .into_iter()
-      .map(|(entry, _)| entry)
```

Comment:
This is a pattern that shows up a lot. This pull request replaces it with

```rust
.clone().into_keys()
```

Reply:
thanks